testing/endpoint.go

```diff
@@ -93,8 +93,16 @@ func (endpoint *Endpoint) CreateClient() (err error) {
 
 		height := endpoint.Counterparty.Chain.LastHeader.GetHeight().(clienttypes.Height)
 		clientState = ibctmtypes.NewClientState(
-			endpoint.Counterparty.Chain.ChainID, tmConfig.TrustLevel, tmConfig.TrustingPeriod, tmConfig.UnbondingPeriod, tmConfig.MaxClockDrift,
-			height, commitmenttypes.GetSDKSpecs(), UpgradePath, tmConfig.AllowUpdateAfterExpiry, tmConfig.AllowUpdateAfterMisbehaviour,
+			endpoint.Counterparty.Chain.ChainID,
```

Comment:
Do we want space reformatting on existing ibc-go files which are otherwise untouched?

Reply:
Mainly asking in case less changes helps with PR approval and less conflicts with other potential changes to endpoint.go

Reply:
good point. I didn't notice the format change